compliance/collection/main.go

```diff
@@ -201,8 +244,14 @@ func main() {
 	ctx = metadata.AppendToOutgoingContext(ctx, "rox-compliance-nodename", getNode())
 
 	stoppedSig := concurrency.NewSignal()
+
 	go manageStream(ctx, cli, &stoppedSig)
 
+	if features.RHCOSNodeScanning.Enabled() {
+		cliNode := sensor.NewComplianceServiceClient(conn)
```

Comment:
Why cannot we reuse the `cli` client from line 241?

Reply:
I've redesigned this area to allow reuse of the existing client